### PR TITLE
How to sign in cloudant account to see dashboard

### PR DESCRIPTION
**Pull request by jannyHou**
Guide users and developers to get access to cloudant dashboard.

### CODE REVIEW

**Comment by jannyHou** [comment]
@superkhau  please check https://github.com/strongloop/strongloop-config/pull/8#issuecomment-255856826 for more context.

Thanks.

File: README.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ There is no free version of local Cloudant to download, so to develop or test wi
     - Click on the Icon, it will direct you to the database page. Check "Service Credentials" on the left to see your credentials.
     - Check "Manage" then click on button "LAUNCH" to see your Cloudant dashboard
 
+For cloudant on both DBaaS and Bluemix, to get access to the cloudant dashboard, you can sign in  https://cloudant.com/sign-in/ with your username and password.
```
Comment (rmg) [inline README.md]: What username and password is this? Do you mean IBM ID, same as was used to log in to Bluemix?

Comment (jannyHou) [inline README.md]: hmm, the description here was unclear, corrected:

> you can sign in  https://cloudant.com/sign-in/ with your **cloudant** username and password.

**Comment by jannyHou** [comment]
cc @superkhau @rmg 
Corrected some description:

- "Limited days free trial" instead of hardcoding the exact number of days. 
- Explicitly tell user to use cloudant username&password to login cloudant dashboard.

**Pull request merged**